pkg/apis/migration/v1alpha1/timed.go

```diff
@@ -0,0 +1,53 @@
+package v1alpha1
+
+import meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// Timed records started and completed timestamps.
+type Timed struct {
+	// Started timestamp.
+	Started *meta.Time `json:"started,omitempty"`
+	// Completed timestamp.
+	Completed *meta.Time `json:"completed,omitempty"`
+}
+
+// MarkReset mark as reset.
+func (r *Timed) MarkReset() {
```

Comment:
is this function really needed? will creating a new `Timed{}` object instead achieve the same result?

Reply:
@alaypatel07 we need this timed.go file as-is for future pipeline improvements. This is taken from virt-controller